sql/updates/world/master/yyyy_mm_dd_xx_world.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE `battleground_template`
+   DROP COLUMN `MinPlayersPerTeam`,
```

Comment:
imho no, better allow small servers to customize requirements.

Reply:
That should still be doable with hotfixes

Reply:
```sql
DELETE FROM `battlemaster_list` WHERE `ID` = 7;
INSERT INTO `battlemaster_list` (`ID`, `Name`, `GameType`, `ShortDescription`, `LongDescription`, `InstanceType`, `MinLevel`, `MaxLevel`, `RatedPlayers`, `MinPlayers`, `MaxPlayers`, `GroupsAllowed`, `MaxGroupSize`, `HolidayWorldState`, `Flags`, `IconFileDataID`, `RequiredPlayerConditionID`, `MapID1`, `MapID2`, `MapID3`, `MapID4`, `MapID5`, `MapID6`, `MapID7`, `MapID8`, `MapID9`, `MapID10`, `MapID11`, `MapID12`, `MapID13`, `MapID14`, `MapID15`, `MapID16`, `VerifiedBuild`) VALUES
(7, 'Eye of the Storm', 'Domination', NULL, NULL, 3, 15, 120, 15, 5, 10, 1, 5, 2851, 24, 525912, 0, 566, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 32638);

DELETE FROM `hotfix_data` WHERE `Id` = 1 AND `TableHash` = 1435332686 AND `RecordId` = 7;
INSERT INTO `hotfix_data` (`Id`, `TableHash`, `RecordId`, `Deleted`) VALUES (1, 1435332686, 7, 0);
```

Will result in an Eye of the storm with max 10 players per team & min 5 players per team

Reply:
With hotfixes the client will also display the correct value
![image](https://user-images.githubusercontent.com/6825243/78472454-2a047d80-7739-11ea-9ca0-4acd57175621.png)

Reply:
then nothing to say :)